build.gradle

```diff
@@ -6,9 +6,9 @@
 buildscript {
     ext {
         opensearch_group = "org.opensearch"
-        opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "3.0.0-alpha1-SNAPSHOT")
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
-        buildVersionQualifier = System.getProperty("build.version_qualifier", "")
+        buildVersionQualifier = System.getProperty("build.version_qualifier", "alpha1")
```

Comment:
Do we need `aplha1` both in version and qualifier?

Reply:
Yes, opensearch_version is to pull from the correct OS Core maven snapshot, combined with qualifier to change the plugin version itself to `3.0.0.0-alpha1-SNAPSHOT` or without snapshot.

Reply:
This is how we did it in 1.x and 2.x.

Reply:
I see what's happening here https://github.com/peterzhuamazon/common-utils/blob/ab13e22e757bf028b7e80fa9ee9b976c07f777ca/build.gradle#L45-L48
Only the first value is being used. Thanks!